indico/modules/rb_new/client/js/components/SpriteImage.jsx

```diff
@@ -77,8 +77,10 @@ class SpriteImage extends React.Component {
         const {pos, width, height, styles, roomsSpriteToken, fillVertical, onClick} = this.props;
         const {contWidth, contHeight} = this.state;
 
+        // Added random string generator to re-fetch sprite
+        const randomId = new Date().getTime();
         const imgStyle = {
-            backgroundImage: `url(${roomsSpriteURL({version: roomsSpriteToken})})`,
+            backgroundImage: `url(${roomsSpriteURL({version: roomsSpriteToken})}?random=${randomId})`,
```

Comment:
this completely breaks caching of the sprite, which is something we really want to avoid.. the sprite token changes when regenerating the sprite so this should be enough to avoid getting outdated cached data